### PR TITLE
Pla 3332 reorganizing the panel page

### PR DESCRIPTION
**Pull request by Biomorphix**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,92 +15,20 @@
...
-            </div>
             <div class="resource-type-toggle">
-                <div class="resource-type audit-res"
-                     value="audit">audit <span></span></div>
-                <div class="resource-type deploy-res"
-                     value="deploy">resources <span></span></div>
+                <div class="resource-type active audit-res"
+                     value="audit">list</div>
                 <div class="resource-type map-res"
-                     value="map">map <span></span></div>
```
Comment (kimslava93) [inline index.html]: Please make sure that we still are able to show warning and error icons, that were set in the spans before

File: index.html [hunk 2]
```diff
@@ -145,12 +73,6 @@
                                 </div>
                             </div>
 
-                            <div class="warning-block warning-note-1">
-                                <span class="error-icon"></span>
-                                <span class="message">Violation checks may be incomplete. <a href="#" id="warning-link"
-                                                                                             class="warning-link">Please review</a> the error and run again to ensure completeness.</span>
-                            </div>
-
```
Comment (enigma91) [inline index.html]: We still need to inform user that one of the resources was failed and redirect user to the resources view

File: js/deploy.js [hunk 3]
```diff
@@ -315,66 +95,32 @@ window.Deploy = (function () {
...
-            }
-            sort(_this.attr('key'), _this.hasClass('desc'));
-        });
-    }
-
     function init(data, sortKey, desc) {
         data = getInitializedCcThisData(data);
         isEnabled = data.isEnabled;
         initGlobalVariables(data);
         initView();
```
Comment (enigma91) [inline js/deploy.js]: this function can be removed, because resources are not displayed anymore

File: js/script.js [hunk 4]
```diff
@@ -192,14 +171,22 @@ $(document).ready(function () {
...
+
+    function setupViewData(isFirstLoad) {
+        var violationCount = auditData.getViolationsCount();
+        var $audit = $('.resource-type-toggle .resource-type.audit-res');
+        if (violationCount) $audit.addClass('alert');
+        $audit.addClass('active')
+            .removeClass('hidden');
+        if (isFirstLoad) $('.audit').addClass('active');
+    }
+
```
Comment (enigma91) [inline js/script.js]: Can you check line:196 setCurrentView() => you can remove this check and always set viewTypes.audit as default

**Pull request merged**